data/SohamRoy-01.json

```diff
@@ -0,0 +1,20 @@
+{
+
+  "name": "Soham Roy",
+  "type": "personal",
+  "bio": "Frontend Developer passionate about full stack and web3 technologies",
+  "avatar":"https://github.com/SohamRoy-01.png",
```

Comment:
hey @SohamRoy-01, you don't need an avatar so can remove it. thanks for contributing!

Reply:
@Hrishi5111998 Ok I removed it....